book/04-git-server/sections/generating-ssh-key.asc

```diff
@@ -37,6 +37,8 @@ d0:82:24:8e:d7:f1:bb:9b:33:53:96:93:49:da:9b:e3 schacon@mylaptop.local
 
 First it confirms where you want to save the key (`.ssh/id_rsa`), and then it asks twice for a passphrase, which you can leave empty if you don't want to type a password when you use the key.
 
+However, if you do use a password, make sure to add the `-o` option. You can also use the `ssh-agent` tool to prevent having to enter the password each time.
+
```

Comment:
Can you split up these sentences onto their own lines? Also, I don't think we need a paragraph break here, so go ahead and delete line 39.